src/main/java/com/denizenscript/depenizen/bukkit/objects/mythicmobs/MythicMobsMobTag.java

```diff
@@ -356,6 +360,32 @@ public static void registerTags() {
         tagProcessor.registerTag(DurationTag.class, "global_cooldown", (attribute, object) -> {
             return new DurationTag(object.getMob().getGlobalCooldown());
         });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.mythic_variable[<name>]>
+        // @returns ElementTag
+        // @plugin Depenizen, MythicMobs
+        // @description
+        // Returns the value of a MythicMobs variable for this MythicMob.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, ElementTag.class, "mythic_variable", (attribute, object, secondVal) -> {
+            return new ElementTag(MythicMobsBridge.getMythicVariable(object.getEntity(), secondVal.asString()), true);
+        });
+
+        // <--[tag]
+        // @attribute <MythicMobsMobTag.mythic_variable_map>
+        // @returns MapTag
+        // @plugin Depenizen, MythicMobs
```

Comment:
missing `@Mechanism MythicMobsMobTag.mythic_variable_map` from this and the other class